packages/stack/contracts-manager/src/index.js

```diff
@@ -62,6 +62,10 @@ export default class ContractsManager {
     })();
   }
 
+  set web3(val) {
+    this._web3 = val;
+  }
+
```

Comment:
Unless we have any means of DI for packages like these, we need to implement a setter here. The reason being that we need to mock the `web3` getter in the test environment. Since it's a getter, we can only assign a value to `contractsManager.web3` which is only possible when there's a setter.

Obviously there's a bunch of issues here:

1. We're accessing `contractsManager.web3` which is supposed to be private
2. We have to introduce a setter just for testing purposes. 

In fact, every module that needs to mock its `web3` suffers from this. Also one downside of using this `async/getter` pattern, and no DI being around.